go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/nwidger/jsoncolor v0.3.2
 	github.com/onsi/ginkgo/v2 v2.11.0
 	github.com/onsi/gomega v1.27.8
-	github.com/openshift-online/ocm-sdk-go v0.1.356
+	github.com/openshift-online/ocm-sdk-go v0.1.367
```

Comment:
We will need to wait the next version as we are changing the BillingModelItem from a struct to a class. 

See https://github.com/openshift-online/ocm-api-model/pull/831/files

Reply:
Used latest sdk version